modules/ROOT/pages/6.0-release-notes-known-issues.adoc

```diff
@@ -0,0 +1,31 @@
+= TinyMCE 6.0 Known issues
+:navtitle: Known issues
+:description: TinyMCE 6.0 known issues
+:keywords: releasenotes, issues
+
+This section documents the single known issue that {productname} 6.0 users may encounter and possible workarounds for this issue.
+
+* xref:known-issues[Known issues]
+
+// tag::known-issues[]
+[[known-issues]]
+== Known issues
+
+There is one known issue affecting {productname} 6.0.
+
```

Comment:
```suggestion
* xref:known-issues[Known issues]

// tag::known-issues[]
[[known-issues]]
== Known issues

There is one known issue affecting {productname} 6.0.

```
I think we could delete these lines, as they're repeating themselves?

Reply:
![doc02](https://user-images.githubusercontent.com/4361382/161883700-86786ec3-af23-4013-9af1-b513757940ca.png)
For reference, this is how it looks at the moment.

Reply:
Realised we are probably using these headers to include this content elsewhere, so removing may break things. @ltrouton you might know?